### DISCUSSION TITLE
Can I update this?

### DISCUSSION THREAD

**Discussion by tommiefr** [nicholas-opensource/OpenWrt-Autobuild#19] [General]
I am using your built for several weeks now and it is rock stable!
Please tell me, in software/updates I see some packages are being updatet.
Can I simply update these packages?

https://imgur.com/jnyNRUJ

Any news on an update?

**Comment by nicholas-opensource** [comment]
I set the packages to be the latest version of 21.02-snapshot when compiled, not the release version. And only the kernel part is the same as the release version. If you continue to follow my future updates, package updates are not necessary, even has a lower version than I released. 

Like busybox 1.33.1-0 to 1.33.1-6, 0 and 6 are just `Autorelease Number`, it is only used to distinguish which time release it. it must be the first time that compiled it without official build-bot, so it got 0. The source code has no fundamental differences. 

However, if you decide not to follow my updates, core/kernel part won't update anymore and this is also the reason why openwrt releases tags. Packages and Luci will continue to get updates from packages-21.02 until openwrt last release of v21.02(Drop support of v21.02), maybe v21.02.8. Besides, upgrading the packages use additional space, i only set rootfs size to 424MiB, maybe you can use `cfdisk` or build in yourself by modifying `SEED/R2S/config.seed` the `CONFIG_TARGET_ROOTFS_PARTSIZE`

**Reply by tommiefr** [reply]
Thank you for your explanation! Really helpful! I will keep using this/your built when you release a new version.
Than I can simply update firmware under system, choose your update file and I am ready to go?

Last built is really stable, over 15 days of continuous use, only one service is not working well with 21.02: bandwidth monitor. I use this one

https://github.com/brvphoenix/luci-app-wrtbwmon

But it is not showing me the real usage. Nothing to do with your build though! The build is great!

**Reply by nicholas-opensource** [reply]
Of course you can upgrade the firmware through the luci page.  But tf card will bring many problems
